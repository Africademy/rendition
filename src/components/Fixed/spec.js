/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import { Fixed, Provider } from '../../../dist'

test('Fixed renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Fixed>Fixed</Fixed>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
