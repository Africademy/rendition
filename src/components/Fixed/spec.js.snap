// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fixed renders correctly 1`] = `
.c0 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c2 {
  position: fixed;
  top: auto;
  right: auto;
  bottom: auto;
  left: auto;
  z-index: 0;
  background: none;
}

.c1 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
}

.c1 h1,
.c1 h2,
.c1 h3,
.c1 h4,
.c1 h5,
.c1 h6,
.c1 button {
  font-family: CircularStd,Arial,sans-serif;
}

<div
  className="c0 c1"
>
  <div
    className="c2 "
  >
    Fixed
  </div>
</div>
`;
